.github/workflows/production-tag.yml

```
@@ -0,0 +1,131 @@
+name: Daily Production Release
+
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: 0 20 * * 1-5
+  # added temporarily to allow GHA to see the action
+  pull_request:
+
+concurrency:
+  group: production-tag
+  cancel-in-progress: true
+
+env:
+  SLACK_CHANNEL: C06DSBT7CBW #status-content-build
+  DSVA_SCHEDULE_ENABLED: true
+
+jobs:
+  create-release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      RELEASE_NAME: ${{ steps.export-release-name.outputs.RELEASE_NAME }}
+
+    steps:
+      - name: Cancel workflow due to DSVA schedule
+        if: ${{ github.event_name == 'schedule' && env.DSVA_SCHEDULE_ENABLED != 'true' }}
+        uses: andymckay/cancel-action@b9280e3f8986d7a8e91c7462efc0fa318010c8b1 # v0.3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef # v2.0.0
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

```suggestion
role-to-assume: {{ vars.AWS_ASSUME_ROLE }}
```

You can remove those lines. OIDC is already configured for next-build. There is a trust setup between Github and AWS that uses and IAM role to grant permissions.